src/OrchardCore/OrchardCore.Mvc.Core/Utilities/StringExtensions.cs

```diff
@@ -236,6 +236,31 @@ public static string RemoveDiacritics(this string name)
             return (sb.ToString().Normalize(NormalizationForm.FormC));
         }
 
+        /// <summary>
+        /// Transforms the culture of a letter to its equivalent representation in the 0-127 ascii table, such as the letter 'é' is substituted by an 'e'
+        /// </summary>
+        /// <param name="s"></param>
+        /// <returns></returns>
+        public static string ReplaceDiacritics(this string s)
+        {
+            string normalizedString = null;
+            StringBuilder stringBuilder = new StringBuilder();
+            normalizedString = s.Normalize(NormalizationForm.FormD);
+            int i = 0;
+            char c = '\0';
+
+            for (i = 0; i <= normalizedString.Length - 1; i++)
+            {
+                c = normalizedString[i];
+                if (CharUnicodeInfo.GetUnicodeCategory(c) != UnicodeCategory.NonSpacingMark)
+                {
+                    stringBuilder.Append(c);
+                }
+            }
+
+            return stringBuilder.ToString().ToLower();
```

Comment:
Should this method do that? Looks like orthogonal to me.

Reply:
You are right I'm removing the ToLower()